server/auth/types/multiple/multi_auth.ts

```diff
@@ -130,6 +130,18 @@ export class MultipleAuthentication extends AuthenticationType {
     return {};
   }
 
+  public async supportsKeepAlive(request: OpenSearchDashboardsRequest): Promise<boolean> {
+    const cookie = await this.sessionStorageFactory.asScoped(request).get();
+    const reqAuthType = cookie?.authType?.toLowerCase();
+
+    if (reqAuthType && this.authHandlers.has(reqAuthType)) {
+      return this.authHandlers.get(reqAuthType)!.supportsKeepAlive(request);
+    } else {
+      // default to true
+      return true;
```

Comment:
can you point to documentation for this default behavior? I think this should be a constant (maybe something dynamically changeable??) and not a hard-coded value. thoughts?

Reply:
Default behavior being that today we are allowing keep alive on all types. Resolving this since changed implementation.